awx/ui/client/lib/components/easy-modal/easy-modal.component.js

```diff
@@ -0,0 +1,29 @@
+const templateUrl = require('~components/easy-modal/easy-modal.partial.html');
```

Comment:
I would probably just name this dialog, as that what it seems to be (a modal with just a simple text body content)

Reply:
It is being used for more than just text content. Its meant to encase anything you want to put in it, including other components.

We use it to display the plugin lookup and input prompt here: https://github.com/ansible/awx/blob/credential_plugins/awx/ui/client/features/credentials/input-source-lookup.partial.html#L1

We also use it for the external test prompt we display when the test button is clicked on an external credential: https://github.com/ansible/awx/blob/credential_plugins/awx/ui/client/features/credentials/external-test-modal.partial.html#L1

With that in mind, would you still like me to rename it?

Reply:
Sounds good to me - I've renamed the component to `dialog.component.js`